keywords/utils.py

```diff
@@ -190,3 +191,13 @@ def check_xattr_support(server_version, sync_gateway_version):
         raise FeatureSupportedError('Make sure you are using Coucbhase Server 5.0+ for xattrs')
     if compare_versions(sync_gateway_version, '1.5') < 0:
         raise FeatureSupportedError('Make sure you are using Coucbhase Sync Gateway 1.5+ for xattrs')
+
+
+def add_cbs_to_sg_config_server_field(cluster_config):
```

Comment:
Can you add a unit test for this function? You can refer to the existing ones under mobile_testkit_tests. Also add some doc string to this function